docs/native-modules-setup.md

```diff
@@ -58,11 +58,11 @@ At this point, follow the steps below to add Windows support to the newly create
 
 ### Updating your package.json
 
-You'll need to ensure you have version 0.62 of both `react-native` and `react-native-windows`. In the directory for your native module project, you can update the dependencies with the following:
+You'll need to ensure you have version 0.63 of both `react-native` and `react-native-windows`. In the directory for your native module project, you can update the dependencies with the following:
 
 ```cmd
-yarn add react-native@0.62 --dev
-yarn add react-native-windows@0.62 --peer
+yarn add react-native@0.63 --dev
+yarn add react-native-windows@0.63 --peer

```

Comment:
I thought @acoates-ms said something about adding a peer dependency locking the module into a particular version of RNW, meaning it would break as the versions rev. Is this true?

Reply:
Yeah, these values should be dependent on what the min version of RNW you want to support.   So this should be something like:
`yarn add react-native-windows@>=0.62 --peer`

Also, if you are working on an existing community project, you can't just update the RN dev version without making changes to the android/ios test projects assuming they have some.  --  You will likely need to do some kind of RN update work flow, depending on what is in the repo.

Reply:
there is a thread on what the scope of validation that we can do is. I'm going to defer creating the guidance for "update/validate other platforms" until that gets settled.

Reply:
Filed https://github.com/microsoft/react-native-windows-samples/issues/244